src/Native/Unix/System.Native/pal_networking.cpp

```diff
@@ -1730,7 +1730,11 @@ extern "C" Error SystemNative_Accept(intptr_t socket, uint8_t* socketAddress, in
 
     socklen_t addrLen = static_cast<socklen_t>(*socketAddressLen);
     int accepted;
+#if defined(HAVE_ACCEPT_4) && defined(SOCK_CLOEXEC)
```

Comment:
Do any of the systems we currently target not have these?  Do any of those currently being brought up by the community (e.g. Android) not have these?